.travis.yml

```diff
@@ -4,28 +4,33 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
 sudo: false
 
 cache:
   directories:
+    - .phpunit
     - $HOME/.composer/cache/files
 
 env:
   matrix:
-    - SYMFONY_VERSION=2.8.* 
+    - SYMFONY_VERSION=3.1.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER=weak
+    - SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_DIR="./.phpunit"
 
 matrix:
   include:
-    - php: 5.6
-      env: DEPS=dev SYMFONY_VERSION=3.1.*
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: DEPS=dev COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.*
+      env: DEPS=dev COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3..*
+    - php: 7.0
+      env: DEPS=dev SYMFONY_VERSION=3.2.*
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION=3.2.*
```

Comment:
indeed but not with 3.2.* of symfony. You and @wouterj said:
- One Symfony Version with each php version
- One PHP Version with each Symfony version

f... you are right and no i understand your both comments
The matrix is "One Symfony Version with each PHP version", so i need to add the other one.